runtime/near-vm-logic/src/logic.rs

```diff
@@ -1743,10 +1748,8 @@ impl<'a> VMLogic<'a> {
         self.check_can_add_a_log_message()?;
         let message = self.get_utf16_string(len, ptr)?;
         self.gas_counter.pay_base(log_base)?;
-        self.gas_counter.pay_per_byte(
-            log_byte,
-            message.encode_utf16().count() as u64 * size_of::<u16>() as u64,
-        )?;
+        // Let's not use `encode_utf16` for gas per byte here, since it's a lot of compute.
+        self.gas_counter.pay_per_byte(log_byte, message.as_bytes().len() as u64)?;
```

Comment:
We don't use utf-16 internally, so once a string is parsed from the utf-16 format, it's now at utf-8 format. Because Rust strings use utf-8 internally.